dataeng/resources/model-transfers.sh

```diff
@@ -28,7 +28,9 @@ fi
 
 ARGS="{mart: ${MART_NAME} }"
 
-dbt deps --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
+source secrets-manager.sh analytics-secure/warehouse-transforms/profiles DBT_PASSWORD
+
+dbt deps --profiles-dir $WORKSPACE/dataeng/profiles --profile $DBT_PROFILE --target $DBT_TARGET
```

Comment:
So the workspace is a jenkins provided environment variable that points to the jenkins job workspace. 
jenkins-job-dsl repo. does not get cloned unless we specify it in the DSL.